docs/docs/cmd/flow/run/run-list.md

```diff
@@ -25,13 +25,18 @@ m365 flow run list [options]
 `--triggerEndTime [triggerEndTime]`
 : Time indicating the exclusive end of a time range of flow runs to return. This should be defined as a valid ISO 8601 string (2021-12-16T18:28:48.6964197Z).
 
+`--asAdmin`
+: Set, to retrieve the results as admin.
+
 --8<-- "docs/cmd/_global.md"
 
 ## Remarks
 
 !!! attention
     This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
 
+By default, the command will try to retrieve Power Automate flow runs you own. If you want to retrieve Power Automate flow runs owned by another user, use the `asAdmin` flag.
```

Comment:
Instead of adding a remark, let's try to include this in the option description.